src/facet_display/Facet.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { useState } from "react"
+import { contains } from "ramda"
+
+import { SearchFacetItem } from "./SearchFacetItem"
+import { BucketWithLabel } from "./FacetDisplay"
+
+const MAX_DISPLAY_COUNT = 5
+const FACET_COLLAPSE_THRESHOLD = 15
+
+interface Props {
+  name: string
+  title: string
+  results: BucketWithLabel[] | null
+  currentlySelected: string[]
```

Comment:
**Minor Suggestion:** I think we should just call these `selected`.

(E.g., for a native `input` element, its value is `input.value`, not `input.currentValue`.)